.github/workflows/release.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - main
   workflow_dispatch:
+  pull_request:
+    paths:
+      - .github/workflows/release.yml
```

Comment:
Nice

Reply:
To give credit, I first saw this here: https://github.com/astral-sh/uv/blob/fbf562d5a86a58d857ed31545f3faed9ed837da7/.github/workflows/build-binaries.yml#L16-L20 - it's great for working on this file, saves so much time going to manually trigger workflows. I think having it trigger on pyproject.toml would be a bit much, but this is an easy win.

Reply:
I think everything is fixed upstream, let's see! :)